coffee/sortable.coffee

```diff
@@ -123,6 +74,66 @@ sortable =
     sortable.typesObject = {}
     sortable.typesObject[type.name] = type for type in types
 
+  sort: (clickedEl, sortedDirection) ->
```

Comment:
I would think about renaming `clickedEl` to `columnHeader`, as when you call this method manually it's not actually the "clicked element".